Author: Steffen Moeller <moeller@debian.org>
Description: Adjusting paths
Disclaimer: This is mostly dead code of upstream.

#Index: boinc/stripchart/samples/get_load
#===================================================================
#--- boinc.orig/stripchart/samples/get_load	2012-01-15 01:31:59.000000000 +0100
#+++ boinc/stripchart/samples/get_load	2012-01-15 02:02:05.000000000 +0100
#@@ -1,11 +1,11 @@
#-#! /bin/csh 
#+#!/bin/sh 
# 
#-set UPTIME = `/usr/bin/uptime | awk '{print $10}' | sed s/,//`
#-if ($UPTIME == "average:") then
#-  set UPTIME = `/usr/bin/uptime | awk '{print $11}' | sed s/,//`
#-endif
#+UPTIME=`uptime | awk '{print $10}' | sed s/,//`
#+if [ "$UPTIME" = "average:"]; then
#+  UPTIME=`uptime | awk '{print $11}' | sed s/,//`
#+fi
# 
#-set CIVDATE = `/bin/date "+%Y:%m:%d:%H:%M"`
#-set UNIXDATE = `/usr/local/bin/perl -e 'print time()'`
#+CIVDATE=`date "+%Y:%m:%d:%H:%M"`
#+UNIXDATE=`perl -e 'print time()'`
# 
# echo $CIVDATE $UNIXDATE $UPTIME
#Index: boinc/stripchart/samples/parse_config
#===================================================================
#--- boinc.orig/stripchart/samples/parse_config	2012-01-15 01:31:59.000000000 +0100
#+++ boinc/stripchart/samples/parse_config	2012-01-15 02:02:05.000000000 +0100
#@@ -1,4 +1,4 @@
#-#!/usr/bin/env python
#+#!/usr/bin/python
# 
# # -*- mode: python; python-indent: 4; -*-
# 
#Index: boinc/stripchart/samples/dir_size
#===================================================================
#--- boinc.orig/stripchart/samples/dir_size	2012-01-15 01:31:59.000000000 +0100
#+++ boinc/stripchart/samples/dir_size	2012-01-15 02:02:05.000000000 +0100
#@@ -1,8 +1,8 @@
#-#! /bin/csh 
#+#!/bin/sh 
# 
#-set SIZE = `du -sk $1 | awk '{print $1*1024}'`
#+SIZE=`du -sk $1 | awk '{print $1*1024}'`
# 
#-set CIVDATE = `/bin/date "+%Y:%m:%d:%H:%M"`
#-set UNIXDATE = `/usr/local/bin/perl -e 'print time()'`
#+CIVDATE=`date "+%Y:%m:%d:%H:%M"`
#+UNIXDATE=`perl -e 'print time()'`
# 
# echo $CIVDATE $UNIXDATE $SIZE
--- a/stripchart/stripchart.cnf
+++ b/stripchart/stripchart.cnf
@@ -1,4 +1,5 @@
-#! /usr/bin/env perl
+
+# This should not be executed, but merely be included
 
 use Time::Local;
 
@@ -6,8 +7,6 @@
 $majorversion = 2; $minorversion = 1;
 
 # Directory where gnuplot is
-# $gnuplot = "/usr/local/gnuplot-3.7";
-#$gnuplot = "/disks/asimov/a/users/hiramc/local/src/gnuplot-3.7";
 $gnuplot = "/usr/bin"
  
 # Temporary files
